rsocket-core/src/commonTest/kotlin/io/rsocket/kotlin/core/RSocketTest.kt

```diff
@@ -192,6 +193,52 @@ class RSocketTest : SuspendTest, TestWithLeakCheck {
         assertTrue(channel.receiveCatching().isClosed)
     }
 
+    @Test
+    fun testStreamInitialUnbounded() = test {
+        val requester = start(RSocketRequestHandler {
+            requestStream {
+                (0..9).asFlow().map {
+                    payload(it.toString())
+                }
+            }
+        })
+        requester.requestStream(payload("HELLO"))
+                .flowOn(PrefetchStrategy(Int.MAX_VALUE, 0))
+                .test {
+                    repeat(10) {
+                        awaitItem().close()
+                    }
+                    awaitComplete()
+                }
+    }
+
+    @Test
+    fun testStreamRequestNUnbounded() = test {
```

Comment:
this test fails, could you check?

Reply:
Didn't run all tests before pushing :facepalm:. I will push the change as a third commit, but I assume all three will be squashed. Or should I squash myself?

Reply:
just do third commit
they will be squashed on merge

Reply:
Pushed the commit :)